src/Loggers/OwnPatternFormatter.cpp

```diff
@@ -3,18 +3,16 @@
 #include <functional>
 #include <IO/WriteBufferFromString.h>
 #include <IO/WriteHelpers.h>
-#include <Common/HashTable/Hash.h>
 #include <Interpreters/InternalTextLogsQueue.h>
-#include <Common/CurrentThread.h>
 #include <base/terminalColors.h>
+#include <Common/CurrentThread.h>
+#include <Common/HashTable/Hash.h>
 
 
-OwnPatternFormatter::OwnPatternFormatter(bool color_)
-    : Poco::PatternFormatter(""), color(color_)
+OwnPatternFormatter::OwnPatternFormatter(bool color_) : Poco::PatternFormatter(""), color(color_)
 {
 }
 
-
```

Comment:
I would suggest changing OwnPatternFormatter.cpp to the way it was before and also updating this branch to the current master. But everything else looks good :)

Reply:
Updated the file and the branch.